test/builder_test.exs

```diff
@@ -40,6 +40,7 @@ defmodule GrpcReflection.BuilderTest do
   end
 
   test "supports all reflection types in proto2" do
+    Protobuf.load_extensions()
```

Comment:
maybe do this in a `setup_all` block?  This is a prereq and would make it more clear that its not actually part of the test